doc/developers/contributing.rst

```diff
@@ -1325,6 +1327,41 @@ interface might be that you want to use it together with model evaluation and
 selection tools such as :class:`model_selection.GridSearchCV` and
 :class:`pipeline.Pipeline`.
 
+Setting `generate_only=True` returns a generator that yields (estimator, check)
+tuples where the check can be called independently from each other, i.e.
+`check(estimator)`. This allows all checks to be run independently and report
+the checks that are failing. In pytest, checks can be run on multiple
+estimators by using `chain.from_iterable`::
```

Comment:
Sorry if that wasn't clear before but I meant that the example with `chain.from_iterable` isn't super useful IMO. And as Andy previously said we should definitly start with the simpler example that only uses 1 estimator.